function.py

```diff
@@ -0,0 +1,6 @@
+
+def function(n):
```

Comment:
Can we have a descriptive name for this function e.g. factorial?
```suggestion
def factorial(n):
```